fedora/generate-dockerfile.sh

```diff
@@ -55,17 +55,29 @@ for repo in $repos; do
 	tiniChecksum="TINI_$repo" && tiniChecksum=$(eval echo \$$tiniChecksum)
 
 	for suite in $suites; do
+
+		if [ $suite == '24' ]; then
+			cgroup='VOLUME ["/sys/fs/cgroup"]'
+			cgroupEntry='mount -t tmpfs -o mode=0755 cgroup /sys/fs/cgroup'
```

Comment:
Why is the cgroup not needed in the other case?

Reply:
We don't need to mount cgroup in some systemd latest versions, it relates to this issue https://github.com/systemd/systemd/issues/6477.